lang_scala/parsing/AST_scala.ml

```diff
@@ -122,7 +122,7 @@ type import_selector = ident_or_wildcard * alias option
 and alias = tok (* => *) * ident_or_wildcard
 [@@deriving show]
 
-type import_expr = stable_id * import_spec
+type import_expr = (ident, stable_id * import_spec) either
```

Comment:
maybe you can add a comment here like (* semgrep-ext: we also support single ident here to accept import $X *)
so it's clear what in the Scala AST are semgrep-specific extensions.